conan/tools/cmake/toolchain/blocks.py

```diff
@@ -941,7 +969,8 @@ def _get_cross_build(self):
                     # cross-build in Macos also for M1
                     system_name = {'Macos': 'Darwin'}.get(os_host, os_host)
                     #  CMAKE_SYSTEM_VERSION for Apple sets the sdk version, not the os version
-                    _system_version = self._conanfile.settings.get_safe("os.sdk_version")
+                    sdk_version = self._conanfile.settings.get_safe("os.sdk_version")
+                    _system_version = self._get_darwin_version(os_host, sdk_version)
```

Comment:
We were previously setting the `os.sdk_version` as `CMAKE_SYSTEM_VERSION` so I converted from that. In some cases the `os.version` is not even set. As I understood, when crosscompiling the `sdk_version` is used instead of the `version`.